src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/interface.py

```diff
@@ -121,14 +125,85 @@ def post_image():
     for file in request.files.to_dict(flat=False)['files']:
         # print(file)
         nparr = np.fromstring(file.read(), np.uint8)
```

Comment:
np.fromstring is deprecated and should be replaced by np.frombuffer

Reply:
True, will fix

Reply:
done